<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitAccountsDialog</class>
 <widget class="QDialog" name="InitAccountsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QComboBox" name="comboBox_field1">
          <item>
           <property name="text">
            <string>Username</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Postigin key</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Email</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Password</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>-</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="radioButton_column">
          <property name="text">
           <string>Column</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Field 2</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>File example:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QRadioButton" name="radioButton_row">
          <property name="text">
           <string>Row</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBox_field2">
          <item>
           <property name="text">
            <string>Postigin key</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Username</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Email</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Password</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>-</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="comboBox_field3">
          <item>
           <property name="text">
            <string>Email</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Username</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Postigin key</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Password</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>-</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Field 3</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="checkBox_skip_empty_lines">
          <property name="text">
           <string>Skip empty lines</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QComboBox" name="comboBox_field4">
          <item>
           <property name="text">
            <string>Password</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Username</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Postigin Key</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Email</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>-</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Field 4</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Choose file</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QPushButton" name="pushButton_choose_file">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>file</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="4">
         <widget class="QTextEdit" name="textEdit_file_example">
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Field 1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
